diode-server/auth/manage_hydra_integration_test.go

```diff
@@ -0,0 +1,108 @@
+//go:build integration_test
+
+package auth_test
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/netboxlabs/diode/diode-server/auth"
+	"github.com/stretchr/testify/require"
+	testcontainers "github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
```

Comment:
Minor: linter rules somehow didn't catch this rule (github.com/netboxlabs/diode to be in the last group) https://github.com/netboxlabs/diode/blob/develop/.github/golangci.yaml#L36-L41